CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 ## Unreleased
 
+## v18.3.3.1
+* Add `auto_pause` and `seconds_until_auto_pause` variables in `rds` module to
```

Comment:
Apologies that this came later but noticed that the wording here is weird.

Reply:
Fixed. Was copy paste issue from cumulus release notes.